op-chain-ops/devkeys/devkeys.go

```diff
@@ -59,6 +59,10 @@ const (
 	SuperchainConfigGuardianKey SuperchainOperatorRole = 1
 	// DependencySetManagerKey is the key used to manage the dependency set of a superchain.
 	DependencySetManagerKey SuperchainOperatorRole = 2
+	// SuperchainProxyAdminOwner is the key that owns the superchain ProxyAdmin
+	SuperchainProxyAdminOwner SuperchainOperatorRole = 3
+	// SuperchainFinalSystemOwner is the key that ownership is transferred to after deployment.
+	SuperchainFinalSystemOwner SuperchainOperatorRole = 4
```

Comment:
What exactly is final system owner? We had this in the original deploy configs, and it was always unclear to me because it doesn't line up with any of the roles we define in the specs, contracts, or docs. Would love to rename this to match the actual contract role, cc @tynes

Reply:
It's a holdover from the legacy deploy-script / config. Happy to rename it to something more accurate.